YourEpicDatabaseDDL.sql

```diff
@@ -12,7 +12,7 @@ CREATE TABLE [ProjTwo].[User] (
 -- Get entire User Table
 SELECT * FROM [ProjTwo].[User];
 
---|--------------------- Add Foreign Key(s) ----------------------------|--
```

Comment:
Is this the only change that was made to the .sql file? Did you just make the changes to the entities themselves?